index.js

```diff
@@ -58,14 +60,24 @@ module.exports = function(options) {
 
   pattern.push('!gulp-load-plugins');
 
+  function logDebug(message, chalkColor) {
+    console.log(chalkColor(message));
+  }
+
```

Comment:
I would move the `if(DEBUG)` into this function:

```
function logDebug(...) {
  if (DEBUG) { console.log(...) }
}
```

Then we don't have to have the `if (DEBUG)` repeated every time we want to log something